### DISCUSSION TITLE
cannot insert a colon (:)

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1869] [Q&A]
Please tell me how to insert a colon (:) into the text

**Comment by sarah-beasley** [comment]
Hi, where are you trying to add a colon please?

**Comment by brockfanning** [comment]
@ukraine-sdg Is the question about using a colon in the content of a YAML file? For example, maybe to get a column into some metadata:

```
MY_FIELD_NAME: This content has a: colon
```

This would definitely fail, because it is not valid YAML. To make this valid, you would need to put some quotes around it, like:

```
MY_FIELD_NAME: "This content has a: colon"
```

Or this:

```
MY_FIELD_NAME: 'This content has a: colon'
```

**Comment by ukraine-sdg** [comment]
Hi, we want to add the colon to the "meta" folder.

**Reply by sarah-beasley** [reply]
Did the above help from Brock? To put ' ' or " " around the text on the right hand side of the colon?

**Comment by ukraine-sdg** [comment]
Thanks for the informative support! Unfortunately, we did not like this
version of displaying the colon, so we refused to use this sign in the text.
